scopes/ui-foundation/ui/create-root.ts

```diff
@@ -74,10 +74,11 @@ ${getImportStatements(defs, 'runtimePath', 'Runtime')}`;
 
 function getImportStatements(aspectDefs: AspectDefinition[], pathProp: string, suffix: string): string {
   return aspectDefs
-    .map(
-      (aspectDef) =>
-        `import ${getIdentifier(aspectDef, suffix, pathProp)} from '${toWindowsCompatiblePath(aspectDef[pathProp])}';`
-    )
+    .map((aspectDef) => {
+      // convert to relative path
+      const url = aspectDef[pathProp].replace(/^.+(?=@teambit)/, '');
```

Comment:
The `aspectDef` doesn't have a relative path from the original source. So I convert it into the relative one in this way. I didn't find any way better to do this yet.

Reply:
You can see this line `const localPath = this.workspace.getComponentPackagePath(component);`
here - 
https://github.com/teambit/bit/blob/7f5c838622d8edef1b07e4aa0a08906ec5c7b182/scopes/workspace/workspace/workspace-aspects-loader.ts#L516

There you will see `const relativePath = this.dependencyResolver.getRuntimeModulePath(component);`
You can use this and add the relative path on the AspectDef as new prop, then use this new prop.